test/630-bridge-vlan.bats

```diff
@@ -140,3 +140,37 @@ echo "$config"
     run_netavark teardown $(get_container_netns_path) <<<"$config1"
     run_netavark teardown $(get_container_netns_path 1) <<<"$config2"
 }
+
+@test "bridge - mtu from default route" {
+    run_in_host_netns ip link add default_route type dummy
+    run_in_host_netns ip link set default_route mtu 9000
+    run_in_host_netns ip addr add 192.168.0.0/24 dev default_route
+    run_in_host_netns ip link set default_route up
+    run_in_host_netns ip route add default via 192.168.0.0
+
+    local vlan=20
+    local config=$(createVlanConfig managed $vlan)
+
```

Comment:
This has nothing to do with the vlan option, I suggest you move the tests into a new bridge mtu file.
Also looking at our tests I actually cannot see any existing bridge mtu config setting test either (only for macvlan/ipvlan). So we should add one as well to make sure the config file does overwrite the default.